testsuite/integration/web/src/test/java/org/jboss/as/test/integration/web/formauth/SecureServlet.java

```diff
@@ -45,13 +43,13 @@
 
     protected void processRequest(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
         Principal user = request.getUserPrincipal();
-        String validateSubject = request.getParameter("validateSubject");
-        if (validateSubject != null && Boolean.valueOf(validateSubject).booleanValue()) {
-            // Assert that there is a valid SecurityAssociation Subject
-            Subject subject = SecurityContextAssociation.getSubject();
-            if (subject == null)
-                throw new ServletException("No valid subject found, user=" + user);
-        }
+//        String validateSubject = request.getParameter("validateSubject");
+//        if (validateSubject != null && Boolean.valueOf(validateSubject).booleanValue()) {
+//            // Assert that there is a valid SecurityAssociation Subject
+//            Subject subject = SecurityContextAssociation.getSubject();
+//            if (subject == null)
+//                throw new ServletException("No valid subject found, user=" + user);
+//        }
```

Comment:
If you make other changes to this PR these lines can be deleted and the commit message changed to 'Remove unused verification dependent on Picketbox."

Both in main and in downstream there's no code that passes requrest parameter "validateSubject" so this is all cruft.